lib/datadog/tracing/configuration/ext.rb

```diff
@@ -12,6 +12,7 @@ module Ext
         ENV_OTEL_TRACES_EXPORTER = 'OTEL_TRACES_EXPORTER'
         ENV_HEADER_TAGS = 'DD_TRACE_HEADER_TAGS'
         ENV_TRACE_ID_128_BIT_GENERATION_ENABLED = 'DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED'
+        ENV_APM_ENABLED = 'DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED' # Temporary until RFC decides final name
```

Comment:
Could we highlight that "temporary" nature in the name? It's a big ask to name it `ENV_APM_ENABLED`, but actually check for standalone feature

_P.S let's do it only if it makes sense (merged before RFC is in final state)_